KVA/Migration.Tool.Source/Services/ModuleLoader.cs

```diff
@@ -25,6 +25,10 @@ public async Task LoadAsync()
                     await sourceInstanceContext.RequestSourceInstanceInfo();
                 }
             }
+            else
+            {
+                logger.LogWarning("The source instance is not configured.");

```

Comment:
Can we make it more specific? E.g. *Source instance query source not detected* or alike